presto-server/pom.xml

```diff
@@ -24,4 +24,41 @@
         <main-class>io.prestosql.server.PrestoServer</main-class>
         <process-name>${project.artifactId}</process-name>
     </properties>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-enforcer-plugin</artifactId>
+                            <executions>
+                                <execution>
+                                    <id>enforce-file-size</id>
+                                    <phase>verify</phase>
+                                    <goals>
+                                        <goal>enforce</goal>
+                                    </goals>
+                                    <configuration>
+                                        <skip>false</skip>
```

Comment:
Is this needed in case `air.check.fail-enforcer` is set? In that case, perhaps we should respect it?